tf_fastmri_data/dataset_builder.py

```diff
@@ -157,10 +176,19 @@ def preprocessed_ds(self):
             self._build_datasets()
         return self._preprocessed_ds
 
+    def _preprocessing_train(self, image, *args):
```

Comment:
Ahh I wanted to ask this to you @zaccharieramzi , I have added a method to just return kspace and images so that we can build on top of this further, or just pickup this raw data (for conventional reconstructions)..
This will remove the assert Not implemented test.. 
We okay with that right?

Reply:
sure